.github/CODEOWNERS

```diff
@@ -329,14 +329,14 @@
 /pkg/collector/corechecks/system/winkmem/         @DataDog/windows-agent
 /pkg/collector/corechecks/system/winproc/         @DataDog/windows-agent
 /pkg/collector/corechecks/systemd/      @DataDog/agent-integrations
-/pkg/collector/corechecks/nvidia/       @DataDog/agent-platform
+/pkg/collector/corechecks/nvidia/       @DataDog/agent-ci-experience
 /pkg/collector/corechecks/windows_event_log/ @DataDog/windows-agent
 /pkg/config/                            @DataDog/agent-shared-components
 /pkg/config/config_template.yaml        @DataDog/agent-shared-components @DataDog/documentation
 /pkg/config/setup/apm.go                @DataDog/agent-apm
 /pkg/config/autodiscovery/              @Datadog/container-integrations
 /pkg/config/env                         @DataDog/container-integrations @DataDog/container-app
-/pkg/config/logs                        @Datadog/agent-shared-components @Datadog/agent-platform
+/pkg/config/logs                        @Datadog/agent-shared-components @Datadog/agent-ci-experience
```

Comment:
Note: I don't think it makes sense for Agent Platform to own this directory at all.

Reply:
```suggestion
/pkg/config/logs                        @Datadog/agent-shared-components
```